src/main/java/uk/gov/justice/hmpps/probationteams/controllers/LocalDeliveryUnitController.kt

```diff
@@ -19,6 +20,7 @@ import uk.gov.justice.hmpps.probationteams.services.LocalDeliveryUnitService
   value = ["local-delivery-units"],
   produces = [APPLICATION_JSON_VALUE],
 )
+@PreAuthorize("hasAnyRole('MAINTAIN_REF_DATA', 'SYSTEM_USER', 'ROLE_SYSTEM_USER')")
```

Comment:
hasAnyRole implicitly adds the `ROLE_` prefix so `SYSTEM_USER` and `ROLE_SYSTEM_USER` are the same

But lets just remove both `SYSTEM_USER` and `ROLE_SYSTEM_USER` from each endpoint - other teams are trying to deprecate it.

Reply:
ok I will just leave MAINTAIN_REF_DATA role.